dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -100,6 +101,10 @@ public final class ZenodoHelper {
     private static String zenodoUrl;
     private static String zenodoClientID;
     private static String zenodoClientSecret;
+    /**
+     * Comma-separated list of GitHub organization names and/or GitHub repo names, e.g., &qout;broadinstitute,DataBiosphere/topmed-workflow-variant-calling&quot;
```

Comment:
This feels like it was written for the String configuration property.

Reply:
I don't follow, please elaborate.

I wanted to make this a YAML array, but backed off because I think AWS Secrets Manager, where this value will be coming from, doesn't do multiline values very well.

That said, looking at YAML again to respond to this, YAML supports arrays defined in a single line, so maybe I should just do that.

```
# Didn't want to do this because of Secrets Manager:
autoGeneratedDoiList:
   - Foo
   - Goo
# Maybe do this:
autoGeneratedDoiList: [Foo, Goo]

Reply:
Nvm, I see my glaring mistake.